timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -144,6 +144,14 @@ types:
         package: com.palantir.atlasdb.backup.api
         fields:
           successful: set<InProgressBackupToken>
+      RefreshBackupRequest:
+        package: com.palantir.atlasdb.backup.api
+        fields:
+          tokens: set<InProgressBackupToken>
+      RefreshBackupResponse:
+        package: com.palantir.atlasdb.backup.api
+        fields:
+          refreshedTokens: set<InProgressBackupToken>
       CompleteBackupRequest:
```

Comment:
the internal backup product will need a higher dependency, yes. unavoidable, I'm afraid, as we need _them_ to have responsibility for refreshing, so that the lock is not leaked if the backup product bounces.